hvantk/tests/ancestry/test_report.py

```
@@ -0,0 +1,448 @@
+
+        output_path = tmp_path / "report.html"
+        result = generate_ancestry_report(mock_ancestry_result, output_path)
+
+        assert isinstance(result, Path)
+
+    def test_report_with_string_path(self, mock_ancestry_result, tmp_path):
+        """Test that function accepts string path."""
+        from hvantk.ancestry.report import generate_ancestry_report
+
+        output_path = str(tmp_path / "report.html")
+        result = generate_ancestry_report(mock_ancestry_result, output_path)
```

Ruff (0.14.14)

[error] Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)